docs/pages/getting-started.md

```diff
@@ -19,7 +19,7 @@ npm i flowbite flowbite-vue
 ```javascript
 module.exports = {
   content: [
-    'node_modules/flowbite-vue/**/*.{js,jsx,ts,tsx}',
+    'node_modules/flowbite-vue/**/*.{js,jsx,ts,tsx,vue}',
```

Comment:
this change was required for custom classes like `last:mb-0` to be visible in projects using `flowbite-vue` (without it custom classes like that was not discovered by Tailwindcss.